ci/do_ci.sh

```diff
@@ -30,6 +30,16 @@ elif [[ "$1" == "cmake.c++20.test" ]]; then
   make
   make test
   exit 0
+elif [[ "$1" == "cmake.legacy.test" ]]; then
+  cd "${BUILD_DIR}"
+  rm -rf *
+  cmake -DCMAKE_BUILD_TYPE=Debug  \
+        -DCMAKE_CXX_FLAGS="-Werror" \
+        -DCMAKE_CXX_STANDARD=11 \
```

Comment:
It doesn't work without the explicit C++11 option?

Reply:
No, gcc4.8 default compilation mode is `-std=c++0x`. We need to make it `-std=c++11` explicitly.